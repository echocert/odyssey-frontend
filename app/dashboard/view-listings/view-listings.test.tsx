import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import { create } from "react-test-renderer";
import ViewListing from "./page";
describe("View Listing Component", () => {
  it("renders the View Listing Component ", () => {
    render(<ViewListing />);
  });

  it("Verify the snapshot of the View Listing Component is matched", () => {
    const wrap = create(<ViewListing />).toJSON();
    expect(wrap).toMatchSnapshot();
  });
});