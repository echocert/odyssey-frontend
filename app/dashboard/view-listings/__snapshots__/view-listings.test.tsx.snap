// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`View Listing Component Verify the snapshot of the View Listing Component is matched 1`] = `
<div
  className="view-listings"
>
  <div
    className="view-listings__header-wrapper"
  >
    <div
      className="view-listings__header-text"
    >
      <label>
        All Properties
      </label>
    </div>
  </div>
  <div
    className="view-listings__section-one"
  />
</div>
`;
